custom_components/powercalc/data/signify/LTP002/model.json

```
@@ -9,5 +9,9 @@
     },
     "name": "Hue ambiance pendant",
```

```suggestion
    "name": "Hue Amaze Suspension Light",
```

Let's also rename to marketed name now we are at it.